### ISSUE TITLE
`at-mixin-pattern` buggy

### ISSUE THREAD

**Issue by evilebottnawi** [kristerkari/stylelint-scss#38]
```json
{
  "rules": {
    "scss/at-mixin-pattern": "^[a-z][a-z0-9-_]*[a-z0-9]$",
  },
  "plugins": [
        "stylelint-scss"
    ]
}
```
```scss
@mixin text-overflow-line-clamp(
    $lines-to-show: 1,
    $background: $body-bg,
    $color: $text-color,
    $line-height: $line-height-base,
    $font-size: $font-size-base
) {
  // Some logic
}
```
```shell
1:1	⚠  Expected @mixin name to match specified pattern (scss/at-mixin-pattern) [stylelint]
```

**Comment by dryoma** [comment]
Thanks for the report. Newlines inside braces were not considered.

@kristerkari how about publishing new patch versions when bugs are fixed?

**Comment by kristerkari** [comment]
@dryoma yes, let's do that.

**Comment by kristerkari** [comment]
@dryoma is there something else to fix for now, or should I release a patch?
